lib/wrappers/contributionReward.ts

```diff
@@ -95,26 +95,26 @@ export class ContributionRewardWrapper extends ContractWrapperBase {
     const externalTokenReward = web3.toBigNumber(options.externalTokenReward);
 
     if (
-      (nativeTokenReward < 0) ||
-      (ethReward < 0) ||
-      (externalTokenReward < 0)
+      (nativeTokenReward.lt(0) ||
```

Comment:
why is this better?

Reply:
the compiler was barfing on `(externalTokenReward < 0)`, i forget exactly why.